manifests/init.pp

```diff
@@ -283,6 +286,7 @@
   Optional[String] $service_config                              = undef,
   Stdlib::Absolutepath $service_logs_dir                        = $rundeck::params::service_logs_dir,
   String $service_name                                          = $rundeck::params::service_name,
+  Boolean $service_restart                                      = $rundeck::params::service_restart,
```

Comment:
can you please add a unit test that sets this parameter and verifies the behaviour?

Reply:
i add a rb spec fileand i rebase my branch